### PR TITLE
Bugfix remove locale warnings and give hint to install VS2010 Redistributables for Windows

### LINKED ISSUE #12: Remove the warnings associated with a non-english locale
I believe these should no longer be necessary and they are irritating to users.

### PR DESCRIPTION
**Pull request by manuelschwarze**
Fixes #12 and #13 .
Added JRebel configuration files in case a developer is using JRebel to speed up development in Eclipse.

### CODE REVIEW

**Comment by greglandrum** [comment]
Iust to confirm: the JRebel changes aren't actually connected to #12 or #13, right? Those are just a bonus?

File: org.rdkit.knime.types/rdkit-chemsrc/org/rdkit/knime/EnvironmentChecker.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * This source code, its documentation and all appendant files
  * are protected by copyright law. All rights reserved.
  *
- * Copyright (C) 2014
```
Comment (greglandrum) [inline org.rdkit.knime.types/rdkit-chemsrc/org/rdkit/knime/EnvironmentChecker.java:6]: When you update a file, you should leave the original copyright in, so please use: 2014-2017

**Review by greglandrum — APPROVED** [review]

**Comment by greglandrum** [comment]
If you can confirm that the jrebel stuff wont' break anything if the developer is not using that, then I'm fine with this being merged after that one copyright line is fixed.

**Comment by manuelschwarze** [comment]
The JRebel stuff is  a "bonus", yes. Would be great, if you could check this pull request out and see if you can start this up properly from your KNIME SDK without having any JRebel plugin installed. I assume so.

**Comment by greglandrum** [comment]
Works fine on my Mac. Merging now.

**Pull request merged**